src/thumbnailer.h

```diff
@@ -88,11 +88,19 @@ class Thumbnailer {
   // GenerateAnimationEqualQuality().
   Status GenerateAnimationEqualPSNR(WebPData* const webp_data);
 
-  // Tries near-lossless to encode each frame. Either
+  // Encodes frames with near-lossless compression, the near-lossless
+  // pre-processing value for each frames can be different. Either
   // GenerateAnimationEqualQuality() or GenerateAnimationEqualPSNR() must be
   // called before to generate animation with lossy encoding. In case of
-  // failure, keeps the latest lossy encoded frame.
-  Status TryNearLossless(WebPData* const webp_data);
+  // failure, returns the latest lossy encoded frames.
+  Status NearLosslessDiffPreProcessing(WebPData* const webp_data);
```

Comment:
Calling it "PreProcessing" while it happens after `GenerateAnimation*()` is a bit confusing